test/testpy/__init__.py

```diff
@@ -40,16 +40,19 @@
 
 class SimpleTestCase(test.TestCase):
 
-  def __init__(self, path, file, arch, mode, context, config, additional=[]):
+  def __init__(self, path, file, arch, mode, context, config, additional=None):
     super(SimpleTestCase, self).__init__(context, path, arch, mode)
     self.file = file
     self.config = config
     self.arch = arch
     self.mode = mode
     self.tmpdir = join(dirname(self.config.root), 'tmp')
-    self.additional_flags = additional
+    if additional is not None:
+      self.additional_flags = additional
+    else:
+      self.additional_flags = []
```

Comment:
Correct but if an empty list is actually passed, we ll still ignore it and create a new list. We don't want that right?